spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/SparkScanBuilder.java

```diff
@@ -114,6 +118,7 @@ public Filter[] pushFilters(Filter[] filters) {
           pushed.add(filter);
         } catch (ValidationException e) {
           // binding to the table schema failed, so this expression cannot be pushed down
+          LOG.warn("binding to the table schema failed, so this expression cannot be pushed down.", e);
```

Comment:
What do you think, "Failed to bind expression to table schema, skipping push down"?

Reply:
Changed. Thanks!